src/module/chat-message/document.ts

```diff
@@ -135,18 +134,18 @@ class ChatMessagePF2e extends ChatMessage<ActorPF2e> {
         if (!actor?.isOfType("character", "npc")) return null;
 
         // Get the strike index from either the flags or the DOM. In the case of roll macros, it's in the DOM
-        const roll = this.rolls.at(0);
-        const strikeIndex = (() => {
-            if (roll instanceof StrikeAttackRoll) return roll.options.strike?.index;
+        const { index, altUsage } = (() => {
```

Comment:
Can this have a return type?

Reply:
I can, but it does mean I'm returning -1 instead of undefined/null. Should be fine though.